deepmd/calculator.py

```diff
@@ -84,10 +84,15 @@ def __init__(
         label: str = "DP",
         type_dict: Optional[dict[str, int]] = None,
         neighbor_list=None,
+        head=None,
```

Comment:
Could you add the new argument into the doc, i.e. after line 47?

Reply:
already done!